test-resources/geo/location-history-data-schema.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Ably Asset Tracking Location History Data",
+  "description": "Describes a sequence of locations over time, as recorded by the Ably Asset Tracking Publisher SDK.",
+  "id": "https://schemas.ably.com/json/asset-tracking-common/LocationHistoryData",
+  "type": "object",
+  "properties": {
+    "version": {
+      "const": 1
```

Comment:
Am I right to think that you've used the `const` here instead of some integer declaration because each new version of the history format will require us to modify this file? 🤔

Reply:
That was my thinking, but I'm not yet really sure how we'd approach version changes, exactly.